lib/internal/modules/esm/load.js

```diff
@@ -1,8 +1,67 @@
 'use strict';
 
 const { defaultGetFormat } = require('internal/modules/esm/get_format');
-const { defaultGetSource } = require('internal/modules/esm/get_source');
 const { validateAssertions } = require('internal/modules/esm/assert');
+const {
+  ArrayPrototypeConcat,
+  RegExpPrototypeExec,
+  decodeURIComponent,
+} = primordials;
+const { getOptionValue } = require('internal/options');
+const { fetchModule } = require('internal/modules/esm/fetch_module');
+
+// Do not eagerly grab .manifest, it may be in TDZ
+const policy = getOptionValue('--experimental-policy') ?
+  require('internal/process/policy') :
+  null;
+const experimentalNetworkImports =
+  getOptionValue('--experimental-network-imports');
+
+const { Buffer: { from: BufferFrom } } = require('buffer');
+
+const fs = require('internal/fs/promises').exports;
```

Comment:
This object is available from user land, it's properties could be mutated. We should use destructuring assignment here.

```suggestion
const { readFile: readFileAsync } = require('internal/fs/promises').exports;
```